rm_fsm:
  robot_type: "standard"
  control_mode: "rc"
  control_param:
    rc_param:
      accel_x: 10.0
      accel_y: 10.0
      accel_angular: 10.0
      coefficient_x: 3.5
      coefficient_y: 3.5
      coefficient_angular: 6.0
      coefficient_yaw: 12.56
      coefficient_pitch: 12.56
      shoot_hz: 5.0
    pc_param:
      accel_x: 10.0
      accel_y: 10.0
      accel_angular: 10.0
      coefficient_x: 3.5
      coefficient_y: 3.5
      coefficient_angular: 6.0
      coefficient_yaw: 12.56
      coefficient_pitch: 12.56
      shoot_hz: 5.0
  power_limit:
    danger_surplus_: 5.0
    roll_back_buffer_: 30.0
    coeff: 0.08
    multiple: 18
